.charts/overwhelm/values.yaml

```diff
@@ -3,10 +3,10 @@ overwhelm:
   deployment:
     image:
       manager:
-        name: ghcr.io/expediagroup/overwhelm
+        repository: ghcr.io/expediagroup/overwhelm
```

Comment:
You also need to update https://github.com/ExpediaGroup/overwhelm/blob/af6f3ff71a5c9f533c56e15738df49ae98db0679/.charts/overwhelm/templates/overwhelm-controller-manager-deployment.yaml#L27

Reply:
good catch, thank you!